src/qt/forms/debugwindow.ui

```diff
@@ -1077,14 +1077,17 @@
                 </widget>
                </item>
                <item row="1" column="0">
-                <widget class="QLabel" name="label_23">
+                <widget class="QLabel" name="peerConnectionTypeLabel">
+                 <property name="toolTip">
+                  <string>The connection type of this peer.</string>
+                 </property>
```

Comment:
bf8135db2a658aed3129ec82e237f8030a8ec7dd

I've noted your [comment](https://github.com/bitcoin-core/gui/pull/162#discussion_r548865476), but here the tooltip provides no additional info. I don't think that adding such a tooltip for the sake itself is a good approach for the GUI:
- https://uxplanet.org/tooltips-in-ui-design-f63e117aa3d1
- https://www.nngroup.com/articles/tooltip-guidelines/

Reply:
True. Perhaps it should link to more information about the connection types (like we have in the codebase), maybe in a markdown file in `/doc`.

Reply:
Maybe I should add more useful info, like in the RPC help (getpeerinfo)

Reply:
Anyway, please keep it useful and _succinct_, or else do not add it at all :)

Reply:
Added info similar to the getpeerinfo help, see updated screenshot in the PR description.